sources/packages/api/src/esdc-integration/disbursement-receipt-integration/disbursement-receipt-files/disbursement-receipt-file-record.ts

```diff
@@ -0,0 +1,47 @@
+import { getDateOnlyFromFormat } from "../../../utilities";
+import { DATE_FORMAT } from "../../models/esdc-integration.model";
+import { DisbursementReceiptRecordType } from "../models/disbursement-receipt-integration.model";
+
+/**
+ * Base class for disbursement receipt record.
+ * All the records of disbursement receipt must be a sub class of this.
+ */
+export class DisbursementReceiptRecord {
+  constructor(
+    protected readonly line: string,
+    protected readonly _lineNumber = 0,
```

Comment:
This can be private if there is an associated public property `get lineNumber()`.

Reply:
The constructor is meant to be used by the sub classes. Hence if it is private, subclass will not be able to use super() and pass the value.